helm/requestor/templates/deployment.yaml

```diff
@@ -104,6 +104,11 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          command: ["/bin/sh"]
```

Comment:
Remind me why this is here?  I recall it might be left over from debugging?

Reply:
Ah great catch, precisely for debugging. Removing now.